crates/sui-adapter-transactional-tests/tests/size_limits/transfered_id_limits_tests.move

```diff
@@ -33,17 +33,16 @@ module Test::M1 {
 }
 
 // transfer below transfer count limit should succeed
-//# run Test::M1::transfer_n_ids --args 1
+//# run Test::M1::transfer_n_ids --args 1 --gas-budget 1000000
```

Comment:
why all these new `gas-budget` options? Why not make it opt-in with a default that existed before.

Reply:
because the I need a much lower default for programmable transactions, which I anticipate being much more common than these limit tests.